runbotics-orchestrator/src/main/java/com/runbotics/domain/UserAuthority.java

```diff
@@ -0,0 +1,26 @@
+package com.runbotics.domain;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Entity
+@Table(name = "jhi_user_authority")
+@IdClass(UserAuthorityId.class)
+public class UserAuthority {
+
+    @Id
+    @Column(name = "user_id")
+    private Long userId;
+
+    @Id
+    @Column(name = "authority_name")
+    private String authorityName;
+
+    public Long getUserId() {
+        return userId;
+    }
+
+    public void setUserId(Long userId) {
+        this.userId = userId;
+    }
+}
```

Comment:
the linking tables should not exist as domains. Create Many-to-Many field in the Authority entity and link there users just like we do with feature keys

Reply:
then just get all guests authorities and you have all users joined to it

Reply:
done